slides-01-intro.qmd

```diff
@@ -0,0 +1,100 @@
+---
+title: "Complex Survey <br>Data Analysis"
+subtitle: "A Tidy Introduction with {srvyr} and {survey}"
+author: 
+  - Stephanie A. Zimmer
+  - Rebecca J. Powell
+  - Isabella C. Velásquez
+institute:
+  - RTI International
+  - Fors Marsh
+  - Posit
+format: revealjs
+date: 2025-08-08
+slide-number: true
+title-slide-attributes: 
+      data-background: "#087e8b"
+      data-background-image: "images/cover.png"
+      data-background-position: 90% center
+      data-background-size: 25%
+      style: "text-align: left; right: 16%"
+---
+
+# Introduction
+
+## About Us
+
+- Stephanie Zimmer, RTI International
+
+- Rebecca Powell, Fors Marsh
+
+- Isabella Velásquez, Posit
+
+::: {.notes}
+Each of us give a brief intro on our experience/background
+:::
+
+## Background {.smaller}
```

Comment:
How much of an introduction into complex survey analysis do we need? Should we assume everybody will know the necessity of weights, etc.?

Reply:
Also, the 'prerequisites' of the course like we have on the workshop page?